.github/actions/java-cache/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Java Cache'
```

Comment:
It is not "java cache".
This action setup the JDK on the current machine.
Could we rename it to avoid confusion in the future?

Reply:
Renamed to `Setup Java` 👍